src/ORM/DatabaseAdmin.php

```diff
@@ -122,8 +124,11 @@ public function build()
         // The default time limit of 30 seconds is normally not enough
         Environment::increaseTimeLimitTo(600);
 
-        // Get all our classes
-        ClassLoader::inst()->getManifest()->regenerate(false);
+        // Flush the manifest if we haven't already
+        $kernel = Injector::inst()->get(Kernel::class);
+        if (!$kernel->getFlushedOnBoot()) {
+            ClassLoader::inst()->getManifest()->regenerate(false);
+        }
```

Comment:
We don’t really need to keep this at all, I’m just being defensive in case anyone wants to run a build programatically like this